src/routes/self-hosted.svelte

```diff
@@ -27,7 +27,7 @@
   contents={{
     title: "Install <div class='whitespace-nowrap inline'>Self-Hosted</div>",
     description: `
-      Install Gitpod Self-Hosted on GKE, EKS, AKS, or K3s.
+      Install Gitpod Self-Hosted on GKE, EKS and AKS.
```

Comment:
```suggestion
      Install Gitpod Self-Hosted on GKE, EKS, and AKS.
```

tiny suggestion for consistency